docker/Makefile

```diff
@@ -0,0 +1,31 @@
+.PHONY: help
+.DEFAULT_GOAL := help
+
+LTS_ALPINE ?= 12-alpine
+
+help: ## Display this help text
+	@grep -E '^[a-zA-Z_-]+[%]?:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+bats: ## Install bats in the project itself
+	@git clone https://github.com/sstephenson/bats.git
```

Comment:
this is scary from a security perspective, should be pinned to a commit / using our fork

Reply:
I'll ensure we use a specific commit hash

Reply:
See https://github.com/elastic/apm-integration-testing/pull/489/commits/a1c8a5f925f7babfa71e6c0a3cd277b422eb3534